src/main/scala/com/github/tkawachi/doctest/DoctestPlugin.scala

```diff
@@ -85,10 +69,10 @@ object DoctestPlugin extends AutoPlugin {
    */
   val doctestGenSettings = Seq(
     doctestTestFramework := (doctestTestFramework ?? ScalaCheck).value,
-    doctestWithDependencies := (doctestWithDependencies ?? true).value,
     doctestDecodeHtmlEntities := (doctestDecodeHtmlEntities ?? false).value,
     doctestMarkdownEnabled := (doctestMarkdownEnabled ?? false).value,
     doctestMarkdownPathFinder := baseDirectory.value * "*.md",
+    testFrameworks            += new TestFramework("utest.runner.Framework"),
```

Comment:
And finally answering your question: no, apparently it does not have any side effect and different flavors of test suites run as expected when we run the scripted tests.